stdlib/public/runtime/CMakeLists.txt

```diff
@@ -102,35 +103,46 @@ if(SWIFT_BUILD_STATIC_STDLIB AND "${sdk}" STREQUAL "LINUX")
     SWIFT_COMPILE_FLAGS ${SWIFT_STANDARD_LIBRARY_SWIFT_FLAGS}
     INSTALL_IN_COMPONENT stdlib)
 
-  foreach(arch IN LISTS SWIFT_SDK_${sdk}_ARCHITECTURES)
-    set(FragileSupportLibrary swiftImageInspectionShared-${SWIFT_SDK_${sdk}_LIB_SUBDIR}-${arch})
-    set(LibraryLocation ${SWIFTSTATICLIB_DIR}/${lowercase_sdk}/${arch})
-    add_custom_command_target(swift_image_inspection_${arch}_static
-      COMMAND
-        "${CMAKE_COMMAND}" -E copy $<TARGET_FILE:${FragileSupportLibrary}> ${LibraryLocation}
-      OUTPUT
-        "${LibraryLocation}/${CMAKE_STATIC_LIBRARY_PREFIX}swiftImageInspectionShared${CMAKE_STATIC_LIBRARY_SUFFIX}"
-      DEPENDS
-         ${FragileSupportLibrary})
-    add_dependencies(stdlib ${FragileSupportLibrary})
-    swift_install_in_component(FILES $<TARGET_FILE:${FragileSupportLibrary}>
-                               DESTINATION "lib/swift_static/${lowercase_sdk}/${arch}"
-                               COMPONENT stdlib)
-  endforeach()
+  set(swift_elf_static_lib_compile_flags ${swift_runtime_library_compile_flags})
+  list(APPEND swift_elf_static_lib_compile_flags -DELF_STATIC_LIB)
+
+  add_swift_target_library(swiftImageInspectionStatic TARGET_LIBRARY STATIC
+    StaticBinaryELF.cpp ImageInspectionELF.cpp
+    C_COMPILE_FLAGS ${swift_elf_static_lib_compile_flags}
```

Comment:
that is: `${swift_runtime_library_compile_flags} -DELF_STATIC_LIB`

Reply:
Fixed.